v2/api/apimanagement/v1api20220801/structure.txt

```diff
@@ -69,7 +194,7 @@ github.com/Azure/azure-service-operator/v2/api/apimanagement/v1api20220801
 │   │   │   ├── HostName: *string
 │   │   │   ├── IdentityClientId: *string
 │   │   │   ├── IdentityClientIdFromConfig: *genruntime.ConfigMapReference
-│   │   │   ├── KeyVaultId: *string
+│   │   │   ├── KeyVaultReference: *genruntime.ResourceReference
```

Comment:
I think this actually _shouldn't_ be a reference because the KV ID isn't a real ARM ID? Ideally we could see an example of what would go here and then we can look at it and confirm. @tomkerkhove might know.

Reply:
This is from the root `Service` - and I believe this is here for when you want the host certificate stored in KV: - 

Here is an example from the QuickStart Templates: - https://github.com/Azure/azure-quickstart-templates/blob/1e5822869895ca5b7f68d51d0209be2ada14e6fb/quickstarts/microsoft.apimanagement/api-management-key-vault-create/azuredeploy.json#L162

Reply:
No longer an ARM reference